tests/unit_tests/test_material.py

```diff
@@ -25,6 +25,40 @@ def test_add_nuclide():
     with pytest.raises(ValueError):
         m.add_nuclide('H1', 1.0, 'oa')
 
+def test_add_elements_or_nuclides():
+    """Test adding multipe elements or nuclides at once"""
+    m = openmc.Material()
+    components = {'H1': 2.0,
+                'O16': (1.0, 'wo'),
+                'Zr': 1.0,
+                'O': (1.0, 'wo'),
+                'U': (1.0, {'enrichment': 4.5}),
+                'Li': (1.0, 'ao', {'enrichment': 60.0, 'enrichment_target': 'Li7'}),
+                'H': (1.0, 'ao', {'enrichment': 50.0, 'enrichment_target': 'H2', 'enrichment_type': 'wo'})}
```

Comment:
```suggestion
    components = {'H1': 2.0,
                  'O16': (1.0, 'wo'),
                  'Zr': 1.0,
                  'O': (1.0, 'wo'),
                  'U': (1.0, {'enrichment': 4.5}),
                  'Li': (1.0, 'ao', {'enrichment': 60.0, 'enrichment_target': 'Li7'}),
                  'H': (1.0, 'ao', {'enrichment': 50.0, 'enrichment_target': 'H2', 'enrichment_type': 'wo'})}
```

This looks fine to me to be honest, but I think some might prefer a uniform indent. I don't think it is worth the CI rerunning to make a change here so no need to worry about this. @paulromano I'm still keen on that pep8 auto formatting :stuck_out_tongue_winking_eye:

Reply:
Luckily, Paul had some comments, so I was able to make the indentation more consistent